### ISSUE TITLE
 Falling trees cause crashes to occur with other mods

### ISSUE THREAD

**Issue by WenXin20** [Shovinus/ChopDown#46]
Forge 14.23.5.2810-1.12.2
Chop Down Updated v1.2.4
[Better Nether](https://minecraft.curseforge.com/projects/betternether) v0.1.8.2
[Cuisine](https://minecraft.curseforge.com/projects/cuisine) v0.5.0-build838

- After breaking the bottom log of the lime tree from Cuisine, I got [this crash](https://gist.github.com/WenXin20/ed1456c8fbeb224a07d02dcaef776a5e) from Better Nether. 
- This seems to have happened because the falling entities got replaced with blocks from Better Nether instead of Cuisine. 
- It doesn't always crash and once the entities land on the ground they are replaced with the correct blocks from Cuisine
- Trees from other mods also can get the falling entities replaced with blocks from other mods and can also crash with that mod.

In this case, the Cuisine leaves and logs were replaced with some type of block from Better Nether along with leaves from Inspirations and some kind of pressure plate:
![image](https://user-images.githubusercontent.com/12000593/52917198-1afd0a80-32ae-11e9-995a-ee492c46ee28.png)

Can anything be done to fix or prevent this?

**Comment by Shovinus** [comment]
Cross talk on block IDs? I am unsure really, how this can occur, the crash itself seems to be caused by the better nether mod trying to do something with what it thinks are it's blocks, but are not, or that get changed back ...

**Comment by WenXin20** [comment]
Yeah, I figured that a block must be getting an invalid state or something. I had another mod crash in a similar way before after cutting down a tree, but I can't remember which one at the moment.

**Comment by WenXin20** [comment]
Is this fixable?
